docs/conf.py

```diff
@@ -0,0 +1,27 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# For the full list of built-in configuration values, see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Project information -----------------------------------------------------
+# https://www.sphinx-doc.org/en/master/usage/configuration.html#project-information
+
+project = 'Ethereum Execution Spec Tests'
```

Comment:
Can we change these values to be similar to this:
https://github.com/ethereum/tests/blob/69c4c2a98dc8a712d4c6f5a817da4f21ff719006/docs/conf.py#L48-L51

Reply:
Yes definitely! Apologies - they where generated automatically through the command line :)